### PR TITLE
[Example] Add Google App Engine example with Node.JS and TypeScript

### PR DESCRIPTION
**Pull request by piotrmoszkowicz**
Example, which shows how to use this action to deploy app to Google App Engine, including example of building process (TypeScript -> JavaScript).

### CODE REVIEW

File: example-workflows/gae/.github/workflows/app-engine.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '275.0.0'
+          service_account_email: ${{ secrets.SA_EMAIL }}
+          service_account_key: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS}}
+
+      # Configure gcloud CLI
+      - name: gcloud Set up
+        run: |
+          gcloud config set project $PROJECT_ID
```
Comment (sethvargo) [inline example-workflows/gae/.github/workflows/app-engine.yml]: You don't need to do this in a separate step anymore. You can set the `project_id` variable on the setup-gcloud action.

File: example-workflows/gae/.github/workflows/app-engine.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      # Setup gcloud CLI
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '275.0.0'
```
Comment (sethvargo) [inline example-workflows/gae/.github/workflows/app-engine.yml]: This is a very old version...

File: example-workflows/gae/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+For pushes to the _default_ branch, `master`, the workflow will:
+
+1. Verify the Google Cloud Platform credentials are correct.
+
+1. Deploy the app to [App Engine](https://cloud.google.com/appengine).
+
+## Setup
+
+1. Create or reuse a Github repository for the example workflow:
```
Comment (sethvargo) [inline example-workflows/gae/README.md]: H in GitHub is capitalized (here and elsewhere)

File: example-workflows/gae/app.yaml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+runtime: nodejs10
+instance_class: F1
```
Comment (sethvargo) [inline example-workflows/gae/app.yaml]: Add newline at EOF

File: example-workflows/gae/tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+		"esModuleInterop": true,
+		"noImplicitReturns": true,
+		"noUnusedLocals": true,
+		"outDir": "dist",
+		"sourceMap": true,
+		"target": "es2017",
+		"baseUrl": "src",
+	},
+	"compileOnSave": true
+}
```
Comment (sethvargo) [inline example-workflows/gae/tsconfig.json]: Add newline at EOF

File: example-workflows/gae/tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (sethvargo) [inline example-workflows/gae/tsconfig.json]: Please use two spaces for indentation.

**Comment by piotrmoszkowicz** [comment]
@sethvargo Done ;)

**Pull request merged**